### ISSUE TITLE
Provide support for storing workspace variables in EMF Simulink models

### ISSUE THREAD

**Issue by imbur** [viatra/massif#85]
Some objects in MATLAB are not part of the Simulink model, but are present in the Matlab workspace. However, these are also necessary to produce a complete model. For example, bus objects represents data flows.

**Comment by imbur** [comment]
One option is to attach a generated `init-script.m` to a model that can be used to create these variables in the workspace when a model is loaded.

**Comment by abelhegedus** [comment]
I agree that these could be modeled in EMF as stored by the importer, but these are not part of the _Simulink_ model and should be defined in a separate Ecore. Also one could argue that the state of the workspace is defined by the user and it is their responsibility to set it up correctly. However, it can be useful to have an "Create workspace model", "Export workspace model to Matlab" functionality separately from the Simulink import/export.

**Comment by imbur** [comment]
I asked Akos @g2k2c8 about this problem, and he pointed out that we are connecting to MATLAB directly. This means that the necessary "workspace initialization" can be done prior to import/export so that we are not going to provide support for storing workspace variables at this point.
